pages/dkp/konvoy/2.2/dkp-upgrade/index.md

```diff
@@ -11,12 +11,300 @@ The DKP upgrade represents an important step of your environment's lifecycle, as
 
 ## Supported upgrade paths
 
-| From Version | To Version |       |       |
-| :----------: | :--------: | :---: | :---: |
-|    1.8.4     |   1.8.5    | 2.1.0 | 2.1.1 |
-|    1.8.5     |   2.1.0    | 2.1.1 |       |
-|    2.1.0     |   2.2.0    |       |       |
-|    2.1.1     |   2.2.0    |       |       |
+<ac:inline-comment-marker ac:ref="4ac92c3d-a7e2-4f18-860e-c7ff2abe0e62">Supported upgrade paths</ac:inline-comment-marker>
```

Comment:
This doesn't work in the preview:

http://docs-d2iq-com-pr-4606.s3-website-us-west-2.amazonaws.com/dkp/konvoy/2.2/dkp-upgrade/#supported-upgrade-paths

![Screen Shot 2022-08-10 at 10 32 31 AM](https://user-images.githubusercontent.com/6485033/183928452-f5b5445c-ea00-4cbe-89f4-b35c780b66bb.png)